packages/backend/src/controllers/statistics.js

```diff
@@ -21,10 +21,10 @@ statisticsRouter.get('/topAccepted/', async (request, response, next, n = 5) =>
     const restaurants = await Restaurant
       .find({}).populate('categories')
     restaurants.sort((a, b) => {
-      if (a.notSelectedAmount === b.notSelectedAmount) {
+      if (a.selectedAmount/a.resultAmount === b.selectedAmount/b.resultAmount) {
```

Comment:
Is this correct? `a.resultAmount` or `b.resultAmount` could be `null`, which would then break this check and the return statement in the `else` block?

Reply:
Prolly fix'd